src/analysis/normed_space/bounded_linear_maps.lean

```diff
@@ -438,3 +438,29 @@ begin
 end
 
 end bilinear_map
+
+/-- Construct a continuous linear map from a linear map that is also an isometry with full range. -/
+def continuous_linear_map.of_isometry (f : E →ₗ[𝕜] F) (hf : isometry f) (hfr : f.range = ⊤) :
+  E ≃L[𝕜] F :=
+linear_equiv.to_continuous_linear_equiv_of_bounds
```

Comment:
Can't this proof be shortened using `continuous_linear_equiv.of_homothety`?

Reply:
Yes, I just pushed a much shorter proof.

Reply:
And added a lemma stating that a linear isometry preserves the norm in the process.